Source/WebInspectorUI/UserInterface/Models/RecordingAction.js

```diff
@@ -115,6 +115,39 @@ WI.RecordingAction = class RecordingAction extends WI.Object
         return typeof propertyDescriptor.value === "function";
     }
 
+    static bitfieldNamesForParameter(type, name, value, index, count)
+    {
+        if (value === 0)
+            return null;
+
+        function test_and_clear_bit(names, value, bit, desc) {
+            if (value & bit)
+                names.push(desc);
+            return value & ~bit;
+        }
+
+        let prototype = WI.RecordingAction._prototypeForType(type);
```

Comment:
we should have a
```
if (!prototype)
    return null;
```
after this because it's not necessarily guaranteed that WebGL(2) is enabled in the Web Inspector page